src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import facebook from "../../images/facebook.svg";
+import instagram from "../../images/instagram.svg";
+import linkedin from "../../images/linkedin.svg";
+
+export const Footer = () => {
+  return (
+    <footer className="flex w-full flex-col bg-wheat p-[80px] gap-[40px] sm:flex-col sm:p-[16px] sm:gap-[26px]">
+      <div className="flex flex-row gap-[287px] sm:gap-[26px] sm:flex-col">
+        <div className="flex flex-col gap-[40px] sm:gap-[26px]">
+          <div className="flex flex-col gap-[24px]">
+            <p className="font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:font-text sm:font-normal sm:leading-[26px] sm:tracking-[-0.02em] sm:text-center ">
```

Comment:
Playing with the code in dev-tools, it seems like tailwind styles that are applied without sm in front of them are also applied when that size is reached. Meaning it behaves like a media query, so there is no need for "sm:font-text sm:font-normal sm:leading-[26px]" because you already applied it earlier in that line without the leading "sm:".
Same thing applies to the entire document.

Reply:
addressed redundant styles